.github/workflows/packages-build-macos-agent.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Macos Wazuh Agent Packages - intel64|arm64
+on:
+  workflow_dispatch:
+    inputs:
+      architecture:
+        type: choice
+        description: Package architecture [intel64, arm64].
+        options:
+          - intel64
+          - arm64
+        required: true
+        default: 'intel64'
+      revision:
+        description: |
+          Package revision (name and metadata).
+          Default is '0'.
+        required: false
+        default: "0"
+      is_stage:
+        type: boolean
+        description: |
+          Should set development/production nomenclature
+          True if package name should have production format.
+          False if package name should have developer format.
+          Default is 'false'.
+        required: false
+        default: false
```

Comment:
These lines can be safely removed as they are already set to false by default. Leaving a comment explaining this (as you've done) is sufficient.